### PR TITLE
Feat: Remove version limit counters

### PR DESCRIPTION
**Pull request by Mikehrn**

### CODE REVIEW

File: packages/frontend-2/components/settings/workspaces/Billing.vue [hunk 1]
```diff
@@ -2,34 +2,16 @@
...
+      <CommonCard class="text-body-xs bg-foundation">
         <p class="text-foreground font-medium">Workspaces are free while in beta.</p>
-        <p class="py-6">
-          Once the beta period ends, workspaces are still free up to
-          {{ versionCount.max }} model versions.
-          <br />
-          To store more versions across your projects you will need to upgrade to a paid
-          plan.
+        <p class="pt-1">
+          Once the beta period ends, your workspace will be converted to a free 30-day
```
Comment (Mikehrn) [inline packages/frontend-2/components/settings/workspaces/Billing.vue:8]: @benjaminvo I had this stashed from last night, I just updated the copy based on what we discussed this morning, but please suggest if you have anything better 😁

File: packages/frontend-2/components/workspace/header/Header.vue [hunk 2]
```diff
@@ -36,13 +36,9 @@
       </div>
     </div>
     <div
-      class="flex justify-between md:items-center gap-x-3 md:flex-row"
-      :class="[isWorkspaceAdmin ? 'flex-col' : 'flex-row items-center']"
+      class="flex justify-between md:items-center gap-x-3 md:flex-row flex-row items-center"
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/components/workspace/header/Header.vue:39]: The responsive classes here seem off. And flex-row can be removed completely.

**Review by andrewwallacespeckle — CHANGES_REQUESTED** [review]
Looks good apart from that 1 minor thing

File: packages/frontend-2/components/settings/workspaces/Billing.vue (hunk 1 above)
Comment (benjaminvo) [reply packages/frontend-2/components/settings/workspaces/Billing.vue:8]: @Mikehrn I think it's pretty good! Minor adjustments:

"After the beta ends, your workspace will start a 30-day free trial. You'll have full access until either the trial expires or you upgrade to a paid plan."

File: packages/frontend-2/components/settings/workspaces/Billing.vue [hunk 3]
```diff
@@ -2,34 +2,16 @@
...
-          <p class="text-foreground-2">
-            Free model versions limit:
-            <span class="text-foreground">{{ versionCount.max }}</span>
-          </p>
-        </div>
       </CommonCard>
 
       <SettingsSectionHeader
-        title="What your workspace bill on a team plan will look like"
+        title="What your workspace bill after the trial period will look like"
```
Comment (benjaminvo) [inline packages/frontend-2/components/settings/workspaces/Billing.vue:14]: Let's write: "What your workspace bill will look like after the trial ends"

**Review by benjaminvo — DISMISSED** [review]

**Review by andrewwallacespeckle — APPROVED** [review]

**Review by andrewwallacespeckle — APPROVED** [review]

**Pull request merged**